channel.py

```diff
@@ -62,6 +67,16 @@ class SaleChannel:
             ('company', '=', Eval('company')),
         ], states=AMAZON_MWS_STATES, depends=['source', 'company']
     ))
+    last_order_import_time = fields.DateTime(
+        'Last Order Import Time', states=AMAZON_MWS_STATES, depends=['source']
+    )
+
+    @staticmethod
+    def default_last_order_import_time():
```

Comment:
Dont think you should put default on this. First time setup should import orders from start!